pkgs/by-name/bu/bugdom2/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, fetchFromGitHub, SDL2, cmake, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  pname = "bugdom2";
+  version = "4.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jorio";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-+Snw5B5DPhzLPxKu9cThO/CNwoffNkSCLat1h1xRdAM=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+  ];
+  buildInputs = [
+    SDL2
+  ];
+
+  installPhase = ''
+    runHook preInstall
+    mkdir -p "$out/bin"
+    mkdir -p "$out/share/Bugdom2"
+    mv Data ReadMe.txt "$out/share/Bugdom2/"
+    install -Dm755 {.,$out/bin}/Bugdom2
```

Comment:
```suggestion
    install -Dm755 Budgom2 -t $out/bin/
```

Reply:
why, a syntax like this was suggested [here](https://github.com/NixOS/nixpkgs/pull/152454#discussion_r775967384) for bugdom1.

Reply:
Why not?
`install -Dm755 Budgom2 -t $out/bin/` is way easier to understand and is not interpreted by Bash.